src/coreclr/gc/unix/cgroup.cpp

```diff
@@ -113,9 +99,9 @@ class CGroup
         if (s_cgroup_version == 0)
             return false;
         else if (s_cgroup_version == 1)
-            return GetCGroupMemoryUsage(val);
+            return GetCGroupMemoryUsage(val, CGROUP1_MEMORY_USAGE_FILENAME);
```

Comment:
Can you remove `s_mem_stat_inactive_file_key_name`/`s_mem_stat_inactive_file_key_length` and pass it as `CGROUP1_MEMORY_STAT_INACTIVE_FIELD`/`CGROUP2_MEMORY_STAT_INACTIVE_FIELD` similar to the other defines?

To get the length you can use `strlen` inside `GetCGroupMemoryUsage`. It is cheap compared to all the other stuff that goes on.

Reply:
@dennis-yemelyanov can you clean this up?

Reply:
yes, sorry, got distracted with another important project, hoping to return to this in a few days and complete this refactoring

Reply:
made a change to remove these fields